src/libs/actions/Task.ts

```diff
@@ -365,8 +365,8 @@ function getOutstandingChildTask(taskReport: OnyxEntry<OnyxTypes.Report>) {
 /**
  * Complete a task
  */
-function completeTask(taskReport: OnyxEntry<OnyxTypes.Report>) {
-    const taskReportID = taskReport?.reportID ?? '-1';
+function completeTask(taskReport: OnyxEntry<OnyxTypes.Report>, reportIDFromAction?: string) {
```

Comment:
```suggestion
function completeTask(taskReport: OnyxEntry<OnyxTypes.Report>) {
```
Do we have a use case that requires using this parameter?  If the change comes from a previous update, I think we can consider remove this prop (the same with the reopenTask function)